tests/upgrades/test_contentview.py

```diff
@@ -279,3 +297,223 @@ def test_post_user_scenario_bug_1429201(self):
         self.assertIsNotNone(container_ids)
         self.assertIn('Error', list(result_fail.values())[0])
         self.assertIn('Complete', list(result_pass.values())[0])
+
+
+class CVUpgradeScenario(CLITestCase):
+    """Test Content-view created before migration gets updated successfully
+    post migration.
+
+        Test Steps:
+
+        1. Before Satellite upgrade:
+        2. Create a product.
+        3. Create custome yum, puppet and file repositories in that product.
+        4. Create content-view.
+        5. Add puppet module, repositories to content view
+        6. Publish content-view
+        7. Upgrade satellite. After Satellite upgrade:
+        8. Remove yum repository which was added to content-view before upgrade.
+        9. Create new yum repository and add it to content-view.
+        10. Remove puppet module which was added to content-view before upgrade.
+        11. Add another puppet module to content-view
+        12. Publish content-view
+     """
+
+    @classmethod
+    def setUpClass(cls):
+        cls.org_name = 'cv_org_name'
+        cls.product_name = 'cv_product_name'
+        cls.yum_repo1_name = 'yum_repo1_name'
+        cls.yum_repo2_name = 'yum_repo2_name'
+        cls.cv_name = 'cv_name'
+        cls.file_repo_name = 'file_repo_name'
+        cls.puppet_module_name = 'versioned'
+        cls.puppet_module_author = 'robottelo'
+        cls.puppet_repo_name = 'puppet_repo_name'
```

Comment:
same as above for all defined name , please refer 

i.e.
https://github.com/SatelliteQE/robottelo/blob/master/tests/upgrades/test_capsule.py#L45
https://github.com/SatelliteQE/robottelo/blob/master/tests/upgrades/test_remoteexecution.py#L47